PostHog/PostHogApi.swift

```diff
@@ -196,6 +196,8 @@ class PostHogApi {
         distinctId: String,
         anonymousId: String?,
         groups: [String: String],
+        personProperties: [String: Any]? = nil,
```

Comment:
```suggestion
        personProperties: [String: Any],
```

can probably be non optional since we will attach default person props anyway